app/views/events/index.html.slim

```diff
@@ -0,0 +1,31 @@
+- content_for :page_header do
+  = render 'layouts/components/page_header',
+           title: 'Events',
+           count: @events.size,
+           links: [ \
+             link_to( \
+               "Add Event",
+               new_event_path,
+               class: "primary button show-for-large" \
+             ) \
+           ]
+
+table
+  thead
+    tr
+      th Name
+      th Description
+      th Facebook Url
+      th Local Group
+  tbody
+    - @events.each do |event|
+      tr
+        td
+          = event.name
+        td
+          = event.description
+        td
+          = link_to event.facebook_url
```

Comment:
Better have the Facebook URL optional. Which means that this will error if `event.facebook_url.nil?`.